ScenesPanel.js

```diff
@@ -217,7 +217,15 @@ function edit_scene_dialog(scene_id) {
 			console.log('setto ' + n + ' a ' + $(this).val());
 		});
 		if(window.CLOUD){
-			window.ScenesHandler.persist_scene(scene_id,true,true);
+			window.ScenesHandler.persist_scene(scene_id,true,true);						
+			setTimeout(function() {

```

Comment:
@Azmoria is it possible to hook into an event instead of an arbitrary timer? I feel like we bump into a few bugs caused by these sorts of race conditions.